crates/oxc_linter/src/rules/eslint/no_useless_call.rs

```diff
@@ -0,0 +1,207 @@
+use oxc_ast::{
+    ast::{CallExpression, ChainElement, Expression, MemberExpression},
+    match_member_expression, AstKind,
+};
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::cmp::ContentEq;
+use oxc_span::Span;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+fn no_useless_call_diagnostic(span: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn("Avoid unnecessary use of .call() and .apply()")
+        .with_help("Replace with a normal function invocation")
+        .with_label(span)
+}
+
+#[derive(Debug, Default, Clone)]
+pub struct NoUselessCall;
+
+declare_oxc_lint!(
+    /// ### What it does
+    /// Disallow unnecessary calls to .call() and .apply()
```

Comment:
```suggestion
    /// Disallow unnecessary calls to `.call()` and `.apply()`
```

Reply:
I’ve incorporated the suggestion.
https://github.com/oxc-project/oxc/pull/8789/commits/1f2d894c730f0bec71db36114e10087dfe5c3000